README.md

```diff
@@ -28,24 +28,28 @@ bugsnag:
 # Usage #
 After the installation the bundle works without any additional settings required, but you can tweak some settings.
 
-## Notify Stages ##
-You can set for which environments you want Bugsnag to get error reports. This is done with the notify_stages setting:
+## Enabled Stages ##
+You can set for which environments you want Bugsnag to be enabled. This is done through the enabled_stages setting:
 
 ```yml
 bugsnag:
-    notify_stages: [development, staging, production]
+    enabled_stages: [dev, prod, staging]
```

Comment:
Perhaps a one-line about where these values comes from (app.php) would be great.